guides/common/modules/proc_increasing-log-levels.adoc

```diff
@@ -0,0 +1,251 @@
+[id="Increasing_Log_Levels_{context}"]
+= Increasing Log Levels
```

Comment:
@maximiliankolb this title?

Reply:
Yeah, all titles. I made two comments: it's basically always `SERVICE TITLE` and `.Changing the log level` which looks weird to me.

Reply:
```suggestion
= Increasing the Logging Levels to Help with Debugging
```

Something like this?

Reply:
Applied.